addon/Feeds/PlacesStatsFeed.js

```diff
@@ -15,6 +17,21 @@ module.exports = class PlacesStatsFeed extends Feed {
       PlacesStatsUpdate(historySize, bookmarksSize)
     ));
   }
+  sendStatsPing() {
+    this.options.metadataStore.asyncGetOldestInsert().then(timestamp => (
```

Comment:
Please add the exception handling code here. i.e. `when asyncGetOldestInsert() resolves with timestamps === null`